server/service/src/missing_program.rs

```diff
@@ -0,0 +1,31 @@
+use repository::{
+    MasterListRow, MasterListRowRepository, ProgramRow, ProgramRowRepository, RepositoryError,
+};
+use util::constants::MISSING_PROGRAM;
+
+use crate::service_provider::ServiceProvider;
+
+// Delete when soft delete for master list is implemented
```

Comment:
perhaps move the comment up - as the whole file won't be required when soft deletes are done
in which case, would also be removed from the `service_provider` too?